tests/Integration/Behaviour/Features/Scenario/Product/Shop/update_multishop_shipping.feature

```diff
@@ -0,0 +1,124 @@
+    And I add a shop "shop2" with name "default_shop_group" and color "red" for the group "default_shop_group"
+    And I add a shop group "shopGroup2" with name "test_second_shop_group" and color "green"
+    And I add a shop "shop3" with name "test_third_shop" and color "blue" for the group "test_second_shop_group"
+    And I add a shop "shop4" with name "test_shop_without_url" and color "blue" for the group "test_second_shop_group"
+    And single shop context is loaded
+    Given I add product "product1" with following information:
+      | name[en-US] | Last samurai dvd |
+      | type        | standard         |
+    And product product1 should have following shipping information:
+      | width                                   | 0       |
+      | height                                  | 0       |
+      | depth                                   | 0       |
+      | weight                                  | 0       |
+      | additional_shipping_cost                | 0       |
+      | delivery time notes type                | default |
+      | delivery time in stock notes[en-US]     |         |
+      | delivery time out of stock notes[en-US] |         |
+      | carriers                                | []      |
+    Given carrier carrier1 named "ecoCarrier" exists
+    And carrier carrier2 named "Fast carry" exists
+    When I update product product1 shipping information with following values:
+      | width                                   | 10.5                 |
+      | height                                  | 6                    |
+      | depth                                   | 7                    |
+      | weight                                  | 0.5                  |
+      | additional_shipping_cost                | 12                   |
+      | delivery time notes type                | specific             |
+      | delivery time in stock notes[en-US]     | product in stock     |
+      | delivery time out of stock notes[en-US] | product out of stock |
+      | carriers                                | [carrier1,carrier2]  |
+    Then product product1 should have following shipping information:
+      | width                                   | 10.5                 |
+      | height                                  | 6                    |
+      | depth                                   | 7                    |
+      | weight                                  | 0.5                  |
+      | additional_shipping_cost                | 12                   |
+      | delivery time notes type                | specific             |
+      | delivery time in stock notes[en-US]     | product in stock     |
+      | delivery time out of stock notes[en-US] | product out of stock |
+      | carriers                                | [carrier1,carrier2]  |
+    And I copy product product1 from shop shop1 to shop shop2
+    Then product product1 should have following shipping information:
```

Comment:
```suggestion
    Then product product1 should have following shipping information for shops "shop1,shop2":
```

Reply:
Also in other PRs I added the copy tests in `shop_management.feature` but since you test it here maybe it's overkill

Reply:
added this.Though it seems that carriers are not copied from shop to shop. Is that expected?Else we should raise an issue and solve it separately probably (idk how big the scope could be)